api/application.ts

```diff
@@ -1,30 +1,26 @@
 import * as bodyParser from 'body-parser'
 import * as cookieParser from 'cookie-parser'
 import * as express from 'express'
-import * as session from 'express-session'
 import * as helmet from 'helmet'
 // import {router as termsAndCRoutes} from './termsAndConditions/routes'
```

Comment:
though this is not part of the changes, can these unuse imports be tidied up

Reply:
we intend to put T&C's back in once it's been deployed to production